hedera-mirror-importer/src/main/resources/db/migration/v2/V2.0.0__create_tables.sql

```diff
@@ -365,13 +365,15 @@ create table if not exists record_file
     count              bigint                 not null,
     digest_algorithm   int                    not null,
     file_hash          character varying(96)  not null,
+    gas_used           bigint default 0,
```

Comment:
This should match v1.
```suggestion
    gas_used           bigint default -1,
```

Reply:
The invariant is that V2 schema should match v1 schema